utils/req-res-log-valiadtor.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python3
```

Comment:
let's add some big comment explaining*
* what it does
* for what purpose
* how to use it